### PR TITLE
Add changing region in dashboard access endpoint

### PR DESCRIPTION
**Pull request by svix-daniel**
Creates a configuration variable for the region in which the dashboard access region can be changed.

### CODE REVIEW

File: server/svix-server/src/cfg.rs [hunk 1]
```diff
@@ -105,6 +105,28 @@ pub struct ConfigurationInner {
     pub api_enabled: bool,
     /// Should this instance run the message worker
     pub worker_enabled: bool,
+
+    #[serde(default = "Default::default", flatten)]
```
Comment (tasn) [inline server/svix-server/src/cfg.rs:109]: Why is this needed?

Comment (svix-daniel) [reply server/svix-server/src/cfg.rs:109]: It was so the structure doesn't have to appear in the default config file, but I'll check how that works with flatten

File: server/svix-server/src/v1/endpoints/auth.rs [hunk 2]
```diff
@@ -25,8 +25,7 @@ async fn dashboard_access(
     let login_key = serde_json::to_vec(&serde_json::json!({
         "appId": app.id,
         "token": token,
-        // Region is unused
-        "region": "eu"
+        "region": cfg.internal.region,
```
Comment (tasn) [inline server/svix-server/src/v1/endpoints/auth.rs:28]: Is it missing an &

**Review by tasn — APPROVED** [review]

**Review by tasn — APPROVED** [review]
